cypress/locators/carousel/index.js

```diff
@@ -1,14 +1,17 @@
 import {
-  SLIDE_INDEX_SELECT, TRANSITION_SELECT, SLIDE, CAROUSEL, PREVIOUS_ARROW_BUTTON,
-  NEXT_ARROW_BUTTON, CAROUSEL_SLIDE_SELECTOR,
+  SLIDE_INDEX_SELECT, TRANSITION_SELECT, CLASSIC_SLIDE, CAROUSEL,
+  CAROUSEL_CONTENT, PREVIOUS_ARROW_BUTTON, NEXT_ARROW_BUTTON, CAROUSEL_SLIDE_SELECTOR,
 } from './locators';
 
 // knobs locators
 export const slideIndexSelect = () => cy.get(SLIDE_INDEX_SELECT);
 export const transitionSelect = () => cy.get(TRANSITION_SELECT);
 
 // component preview locators
-export const slide = () => cy.iFrame(SLIDE);
+export const classicSlide = () => cy.iFrame(CLASSIC_SLIDE);
+export const slide = i => cy.iFrame(CAROUSEL_CONTENT)
```

Comment:
If you use locator which I proposed below you can use this
```export const slide = index => cy.iFrame(DEFAULT_SLIDE).eq(index);```

Reply:
Thanks, I'll use this solution (with the `DEFAULT_` prefix for the reason stated above) 👍